app/src/main/assets/ime/text/symbols/neo2.json

```
@@ -0,0 +1,46 @@
+{
+  "type": "symbols",
+  "name": "neo2",
+  "label": "Neo2",
+  "authors": [ "ostrya" ],
+  "direction": "ltr",
+  "modifier": "neo2",
+  "arrangement": [
+    [
+      { "code": 8230, "label": "…" },
+      { "code":   95, "label": "_" },
+      { "code":   91, "label": "[" },
+      { "code":   93, "label": "]" },
+      { "code":   94, "label": "^" },
+      { "code":   33, "label": "!" },
+      { "code":   60, "label": "<" },
+      { "code":   62, "label": ">" },
+      { "code":   61, "label": "=" },
+      { "code":   38, "label": "&" },
+      { "code":  383, "label": "ſ" }
+    ],
+    [
+      { "code":   92, "label": "\\" },
+      { "code":   47, "label": "/" },
+      { "code":  123, "label": "{" },
+      { "code":  125, "label": "}" },
+      { "code":   42, "label": "*" },
+      { "code":   63, "label": "?" },
+      { "code":   40, "label": "(" },
+      { "code":   41, "label": ")" },
+      { "code":   45, "label": "-" },
+      { "code":   58, "label": ":" },
+      { "code":   64, "label": "@" }
+    ],
+    [
+      { "code":   35, "label": "#" },
+      { "code":   36, "label": "$" },
```

```suggestion
      { "code": -801, "label": "currency_slot_1" },
```

The user has the ability to set the currency set for the symbols layout, thus neo2 should also respect this. The real currency symbol will then automatically be inserted in runtime.

---

Makes sense to use this feature since it's there. But from the layout point of view, I feel the Euro sign below the numerical 7 would be more suited. The $ sigh here is I think more meant for programming purposes.